### PR TITLE
Feature/mna cleanup

### PR DESCRIPTION
**Pull request by jrddunbr**
Cleaning up some MNA code to make it easier to use.

TODO: Fix some of the thermal code a bit more. Some of those variables still use names that are unintelligible.. thus why I haven't changed them yet.

Build result: https://teamcity.vars.ja4.org/buildConfiguration/ElectricalAge_Build/10?hideProblemsFromDependencies=false&hideTestsFromDependencies=false&expandBuildChangesSection=true

### CODE REVIEW

**Comment by jrddunbr** [comment]
Build passes for commit 0a73f72

File: src/main/java/mods/eln/entity/ReplicatoCableAI.java [hunk 1]
```diff
@@ -134,7 +134,7 @@ public void updateTask() {
     }
 
     boolean isElectricalCableInterresting(ElectricalCableElement c) {
-        if (c.descriptor.signalWire || c.electricalLoad.getU() < 30) {
+        if (c.descriptor.signalWire || c.electricalLoad.getVoltage() < 30) {
```
Comment (Caeleron) [inline src/main/java/mods/eln/entity/ReplicatoCableAI.java:137]: Replace with
```java
return !(c.descriptor.signalWire) && c.electricalLoad.getVoltage() >= 30
```

**Review by Caeleron — CHANGES_REQUESTED** [review]
Other than removing the unnecessary if-statement, looks good to me.

File: src/main/java/mods/eln/entity/ReplicatoCableAI.java (hunk 1 above)
Comment (jrddunbr) [reply src/main/java/mods/eln/entity/ReplicatoCableAI.java]: Fixed.

**Pull request merged**